pwndbg/file.py

```diff
@@ -134,3 +135,18 @@ def readlink(path):
     data = data[:-len(ending)]
 
     return data
+
+def file(path):
```

Comment:
Having this module named `pwndbg.file` might be misleading.  Since this and `pwndbg.checksec` are both wrappers around command-line utilities, perhaps they might be under e.g. `pwndbg.wrappers.file` and `pwndbg.wrappers.checksec` rather than second-level modules.